arches/app/templates/views/components/widgets/reference-select.htm

```diff
@@ -0,0 +1,58 @@
+{% extends "views/components/widgets/base.htm" %}
+{% load i18n %}
+
+{% block form %}
+<div class="row widget-wrapper" data-bind="class: nodeCssClasses">
+    <div class="form-group">
+        <span class="control-label widget-input-label" data-bind="text:label, class: nodeCssClasses"></span>
+        <!-- ko if: node -->
+        <i data-bind="
+            css: {'ion-asterisk widget-label-required': node.isrequired},
+            attr: {'aria-label': $root.translations.required}        
+        "></i>
+        <!-- /ko -->
+        <div class="col-xs-12 resource-instance-wrapper" data-bind="class: nodeCssClasses">
+            <select style="display:inline-block;"
+                data-bind="
+                    disable: false,
+                    select2Query: {
+                        select2Config: select2Config
+                    },
+                    attr: {'data-label': label, 'aria-label': label}
+                ">
+        </div>
+    </div>
+</div>
+{% endblock form %}
+
+{% block config_form %}
+<div>
+    <div class="control-label">
+        <span data-bind="text: $root.translations.placeholder"></span>
+    </div>
+    <div class="col-xs-12 pad-no crud-widget-container">
+        <input
+            class="form-control input-md widget-input"
+            data-bind="
+                attr: {placeholder: $root.translations.placeholder, 'aria-label': $root.translations.placeholder},
+                value: placeholder,
+                valueUpdate: 'keyup'
+            "
+        >
+    </div>
+</div>
+{% endblock config_form %}
+
+{% block report %}
+<dt data-bind="text: label, class: nodeCssClasses"></dt>
+<!-- ko foreach: value -->
+    <dd data-bind="class: $parent.nodeCssClasses">
+        <a data-bind="attr:{'href': uri, 'target': '_blank'}, text: $parent.getPrefLabel(labels), style: {'color': 'steelblue'}"></a>
```

Comment:
This link doesn't seem to work when the report is viewed from Search Results > Details. Have you seen this before?

Reply:
This is really puzzling. I added a click handler to that element and clicks are landing as expected. It must be related to the css of the summary(details) report. I took a look at the url datatype and see the same behavior.

Reply:
Thanks for looking into it #10639